README.md

```diff
@@ -1,51 +1,110 @@
+* either click on the link provided in the description of an epic, or select the appropriate template when clicking
+  the "new issue" button,
+* change the default title to a meaningful and concise one,
+* provide the details required by the respective template.
+
+#### [Organizational Issues](https://github.com/nfdi-de/section-metadata-wg-onto/issues?q=is%3Aissue+is%3Aopen+label%3Aorganizational)
+Organizational issues are those that are concerned **only** with "meta" discussions about how to structure our work.
+Hence, they should not be associated with any epic or epic subtask issue. 
+For example, when you have questions around or problems with the use of this repository, you should be filing an 
+organizational issue. 
+To file a new organizational issue, please use [this template](https://github.com/nfdi-de/section-metadata-wg-onto/issues/new?assignees=&labels=organizational&projects=&template=organizational-issue.md&title=organizational%3A+%5BADD+YOUR+CONCISE+ISSUE+TITLE+HERE%5D), change its default title and add a meaningful 
+description about what needs to be discussed.
+
+There exists one permanently open, organizational issue that is pinned to the top of the issue list. [This issue](https://github.com/nfdi-de/section-metadata-wg-onto/issues/12)
+explains how all open epic subtask issues can be prioritized by up-voting. This is intended to to be discussed in 
+upcoming calls.
+
+## Repository Interaction Workflow
+We expect the WG members to follow the activity of this repository by starring or watching it. Members who want to 
+contribute directly must either be made part of the repository team (contact [@StroemPhi](https://github.com/StroemPhi)), 
+or fork the repository.
+
+A prototypical interaction of a WG member with this repository would entail:
+* checking [here](https://github.com/nfdi-de/section-metadata-wg-onto/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc+-label%3A%22charter+epic%22+-label%3A%22organizational%22+) regularly, to see if any new open epic subtask issues have been created to be voted on,
+* creating epic subtask issues,
+* commenting on issues (participate in discussions),
+* creating an issue branch when working on a specific subtask that requires the creation or editing of files,
+* submitting changes made on an issue branch via a pull request.
+
+### Using GitHub Project to manage issues
+TODO: explain how to interact with our [project](https://github.com/orgs/nfdi-de/projects/1).
+
+## Repository Structure
+
+* [mapping](mapping) - is the workspace for charter epic #7
+  * contains a [README.md](mapping%2FREADME.md) to document the completed work
+* [TLO_Infos](TLO_Infos) - is the workspace for charter epic #3
+  * TODO: add README for this epic and possibly rename the folder to something more suitable
+* TODO: add folders and README for the other epics
+* [Meetings](Meetings) - contains the notes of our previous meeting notes in Markdown format
 
 ## Important Links
```

Comment:
Instead of listing all links as sepearte section I suggest to create an "output" (or "deliverables") section just after the intro where we linked outputs like the wg charter. The links regarding the contribution process could be move into the contribution (now still "Prerequisites") section.

Reply:
Hopefully improved accordingly in [3f3c950](https://github.com/nfdi-de/section-metadata-wg-onto/pull/16/commits/3f3c9500c85fb01c79a59415098334f60b427683)